lib/src/builder/execute.rs

```diff
@@ -340,6 +354,10 @@ impl TxExecStrategy for TkoTxExecStrategy {
             print_duration("Tx transact time: ", tx_transact_duration);
             print_duration("Tx misc time: ", tx_misc_duration);
         }
+        clear_line();
+        print_duration("Tx transact time: ", tx_transact_duration);
+        print_duration("Tx misc time: ", tx_misc_duration);
+        println!("actual Tx: {}", actual_tx_no);
```

Comment:
Mostly duplicated from the printing above which is disable in optimistic mode, so I think should be removed